nixos/modules/services/networking/minidlna.nix

```diff
@@ -66,7 +96,37 @@ in
 
     services.minidlna.config = mkOption {
       type = types.lines;
-      description = "The contents of MiniDLNA's configuration file.";
+      description =
+      ''
+        The contents of MiniDLNA's configuration file.
+        When the service is activated, a basic template is generated
+        from the current options opened here.
+      '';
+    };
+
+    services.minidlna.extraConfig = mkOption {
+      type = types.str;
```

Comment:
`types.lines`?

Reply:
This seems to be fine with types.lines as well, fixing.